test/test-pages/ehow-2/expected.html

```diff
@@ -0,0 +1,227 @@
+<p>Avoid pre-packaged foods and pre-made deli platters. These can be quite costly. Instead, make your own cheese and deli platters for less than half the cost of pre-made.</p></span></div>
+<figure class="stepThumb">
+    <img src="http://img-aws.ehowcdn.com/640/cme/cme_public_images/www_ehow_com/cdn-write.demandstudios.com/upload/image/D0/51/B6AED06C-5E19-4A26-9AAD-0E175F6251D0/B6AED06C-5E19-4A26-9AAD-0E175F6251D0.jpg" alt="Cost effective appetizers are just as satisfying as pre-made deli platters." class="photo" data-credit="Mark Stout/iStock/Getty Images" data-pin-ehow-hover="true" data-pin-no-hover="true" />
+</figure>
+<figcaption class="small caption">
+    Mark Stout/iStock/Getty Images </figcaption>
+</div>
+</div>
+</span>
+</span>
+<span>
+<span>
+<div class="mod step"><div class="stepContent mod"><div class="content"><span><p>Instead of an evening dinner party, host a grad lunch or all appetizers party. Brunch and lunch fare or finger food costs less than dinner. Guests also tend to consume less alcohol in the middle of the day, which keeps cost down.</p></span></div>
+<figure class="stepThumb">
+    <img src="http://img-aws.ehowcdn.com/640/cme/cme_public_images/www_ehow_com/cdn-write.demandstudios.com/upload/image/35/B4/DD5FD05A-B631-4AFE-BC8F-FDACAD1EB435/DD5FD05A-B631-4AFE-BC8F-FDACAD1EB435.jpg" alt="A brunch gathering will cost less than a dinner party." class="photo" data-credit="Mark Stout/iStock/Getty Images" data-pin-ehow-hover="true" data-pin-no-hover="true" />
+</figure>
+<figcaption class="small caption">
+    Mark Stout/iStock/Getty Images </figcaption>
+</div>
+</div>
+
+
+</span>
+</span>
+<span>
+<span>
+<div class="mod step"><div class="stepContent mod"><div class="content"><span><p>Decorate your party in the graduate's current school colors or the colors of the school he or she will be headed to next. Décor that is not specifically graduation-themed may cost a bit less, and any leftovers can be re-used for future parties, picnics and events.</p></span></div>
+<figure class="stepThumb">
+    <img src="http://img-aws.ehowcdn.com/640/cme/cme_public_images/www_ehow_com/cdn-write.demandstudios.com/upload/image/A1/FA/2C368B34-8F6A-45F6-9DFC-0B0C4E33FAA1/2C368B34-8F6A-45F6-9DFC-0B0C4E33FAA1.jpg" alt="Theme the party by color without graduation-specific decor." class="photo" data-credit="jethuynh/iStock/Getty Images" data-pin-ehow-hover="true" data-pin-no-hover="true" />
+</figure>
+<figcaption class="small caption">
+    jethuynh/iStock/Getty Images </figcaption>
+</div>
+</div>
+</span>
+</span>
+
+
+<h2 class="RsTitle head mg-2 title">
```

Comment:
Ideally we shouldn't include this either, but I don't know off-hand how best to do this, so let's leave it for now.